prow/github/client.go

```diff
@@ -2721,8 +2722,9 @@ func (c *Client) GetColumnProjectCard(columnID int, cardNumber int) (*ProjectCar
 	if err != nil {
 		return nil, err
 	}
+
 	for _, card := range cards {
-		if card.ContentID == cardNumber {
+		if strings.Compare(card.ContentURL, issueURL) == 0 {
```

Comment:
Why is `strings.Compare` being used instead of the `==` operator?

Reply:
true, fixed.